Traceback (most recent call last):
  File "/home/aayush/CS327 Compilers/Cobra/helper.py", line 25, in <module>
    main()
  File "/home/aayush/CS327 Compilers/Cobra/helper.py", line 20, in main
    abt = resolve(ast)
  File "/home/aayush/CS327 Compilers/Cobra/resolver.py", line 30, in resolve
    er = resolve(e, env, fresh)
  File "/home/aayush/CS327 Compilers/Cobra/resolver.py", line 28, in resolve
    return Var(x, lookup(env, x))
  File "/home/aayush/CS327 Compilers/Cobra/resolver.py", line 8, in lookup
    raise ValueError("No value found.")
ValueError: No value found.
